docs/IPFS.md

```diff
@@ -0,0 +1,67 @@
+# IPFS
+For an overview about IPFS, visit the [IPFS project site](https://ipfs.tech/).
+
+In IPFS there are two protocols. IPFS and IPNS (their workings are explained in detail [here](https://docs.ipfs.tech/concepts/)). The ideal way to access data on the IPFS network is trough those protocols. For example to access the Big Buck Bunny video the ideal way to access it is like: `ipfs://bafybeigagd5nmnn2iys2f3doro7ydrevyr2mzarwidgadawmamiteydbzi`
+
+## IPFS and IPNS protocol handling
+There are various ways to access data from the IPFS network. One such way is through the concept of public "[gateways](https://docs.ipfs.tech/concepts/ipfs-gateway/#overview)". The short version is that entities can offer gateway services. An example here that is hosted by Protocol Labs (who also makes IPFS) is dweb.link and ipfs.io. Both sites expose gateway functionality. Getting a file through ipfs.io looks like this: https://ipfs.io/ipfs/bafybeigagd5nmnn2iys2f3doro7ydrevyr2mzarwidgadawmamiteydbzi
+
+If you were to be [running your own IPFS node](https://docs.ipfs.tech/how-to/command-line-quick-start/) then you, by default, also have a [local gateway](https://docs.ipfs.tech/concepts/ipfs-gateway/#gateway-providers) running. In it's default configuration the earlier example would then also work in this link: http://127.0.0.1:8080/ipfs/bafybeigagd5nmnn2iys2f3doro7ydrevyr2mzarwidgadawmamiteydbzi
```

Comment:
no backticks

Reply:
? There is no backtick in the section/line you comment on. The only one i saw (and removed locally) is the one on line 4.

Or do you mean that all "code blocks" that have \``` and end with \``` should all be removed?

Reply:
I mean you should write the URL within backticks:
`http://127.0.0.1:8080/ipfs/bafybeigagd5nmnn2iys2f3doro7ydrevyr2mzarwidgadawmamiteydbzi`

Reply:
Right, fixed all of them.